app/Database/16-07-27-weekly-summary.txt

```diff
@@ -0,0 +1,22 @@
+16-07-27 Weekly summary (brief).
```

Comment:
@klmerryn you shouldn't use a pull request to share status updates.

It wouldn't be so bad, but in particular, the contents of this document have nothing to do with this pull request. This pull request is in the process of being wrapped up, and which I hope to merge in soon. But I can't do so with this file in here.